std/typecons.d

```diff
@@ -768,6 +770,12 @@ if (distinctFieldNames!(Specs))
             return field[] == rhs.field[];
         }
 
+        bool opEquals(R...)(R rhs) const
+        if (R.length > 1 && areCompatibleTuples!(typeof(this), Tuple!R, "=="))
```

Comment:
Does `areCompatibleTuples` also include the length check?

Reply:
Yes.